neo/Ledger/MemoryPool.cs

```diff
@@ -253,6 +255,34 @@ public IEnumerable<Transaction> GetSortedVerifiedTransactions()
             }
         }
 
+        // This function return some statistics about the best _maxTxPerBlock and _maxLowPriorityTxPerBlock txs of the mempool
+        public JObject GetMemPoolInfoForNextBlock()
+        {
+            int minSizeSortedHP = Math.Min(_maxTxPerBlock, _sortedHighPrioTransactions.Count);
```

Comment:
This method is misleading if the SimplePolicy plugin isn't loaded since it uses the full count of transactions in that case to create the statistics.

Reply:
Also different nodes could have different policy for how many transactions to allow each block. Even the consensus nodes themselves don't have to validate the same amount of transactions each block. So I think this call will be misleading.

Reply:
Perhaps it should always just use the full number of transactions in the pool.

Reply:
In general, different policies will be reality for CrossChains, Private Chains, such as Ont.

Inside the NEO core chain those values should be the same in most cases.

Reply:
Abou the SimplePolicy not loaded we need to double check and discuss the best strategy.

Reply:
Regarding the SimplePolicy not loading, it can be a note such as the ApplicationLog.